org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/devicedata.json

```diff
@@ -18,7 +17,7 @@
             "last_reading" : {
                 "max_set_point" : 100,
                 "min_set_point" : 0,
-                "units" : "C",
+                "units" : { "temperature": "c" },
```

Comment:
I looked at the value that is returned by an actual device and it was lowercase.

---
In reply to: [101429526](https://github.com/openT2T/translators/pull/163#discussion_r101429526) [](ancestors = 101429526)